system/HTTP/Files/UploadedFile.php

```diff
@@ -267,6 +276,15 @@ public function getClientName(): string
         return $this->originalName;
     }
 
+    /**
+     * (PHP 8.1+)
+     * Returns the webkit relative path of the uploaded file on directory uploads.
+     */
+    public function getClientPath(): string|null
```

Comment:
use nullable instead

```suggestion
    public function getClientPath(): ?string
```